docs/discord-social-sdk/development-guides/using-provisional-accounts.mdx

```diff
@@ -498,6 +498,28 @@ def unmerge_provisional_account(external_auth_token):
 If you have a server backend, you'll want to use the server-to-server unmerge endpoint rather than the SDK helper method to maintain better security and control over the unmerge process.
 :::
 
+#### Unmerging with Bot Token Endpoint
+
+If you're using the [Bot Token Endpoint](/docs/discord-social-sdk/development-guides/using-provisional-accounts#server-authentication-with-bot-token-endpoint) for authentication, you can unmerge accounts without an external auth token.
```

Comment:
Do we want to mention that you can use this to unmerge even if you've lost the discord auth token?

Something like:


```
:::info
This endpoint can also be useful in cases where the Discord Auth token has been lost to to error or data loss, and an unmerge operation is required to migrate to a provisional account before re-linking a Discord account.
:::
```

WDYT?